spec/controllers/pxe_controller/iso_datastores_spec.rb

```diff
@@ -0,0 +1,15 @@
+describe PxeController do
+  before(:each) do
+    set_user_privileges
+  end
+
+  describe "#iso_datastore_set_form_vars" do
+    it 'assigns form options' do
+      FactoryGirl.create(:ems_redhat)
+      controller.instance_variable_set(:@isd, FactoryGirl.create(:iso_datastore))
+      controller.send(:iso_datastore_set_form_vars)
+
+      expect(assigns(:edit)[:emses]).not_to be_empty
```

Comment:
I think this should equal the `[[name, id]]` from the `:ems_redhat` that was created above, right?  I'm not a fan of `.not_to be_empty` when we know exactly what it should equal.